src/core/stac/qgsstaccontroller.cpp

```diff
@@ -223,6 +223,30 @@ QgsStacItemCollection *QgsStacController::fetchItemCollection( const QUrl &url,
   return ic;
 }
 
+QgsStacCollections *QgsStacController::fetchCollections( const QUrl &url, QString *error )
```

Comment:
Could you please add a simple auto test for this function?

There seem to be some sample data here https://github.com/radiantearth/stac-api-spec/tree/main/fragments/collectioncollection/examples - even though they were added post-1.0.0 spec release, they seem like omissions from the original spec rather than something new...

Reply:
Added tests for `fetchItemCollection()` and `fetchCollections()`.
The term `collectionCollection` is used in the above linked examples :)